### DISCUSSION TITLE
How to I exit cog while running on DRM?

### DISCUSSION THREAD

**Discussion by chrisvollorenew** [Igalia/cog#493] [Q&A]
Hello, 

I am running cog on drm (cog -P drm file:/// ..  ) mode on an embedded device.
The cog browser launches from a process that is called at boot.
If I plug in a keyboard to my embedded device, I am able to navigate the browser, but I am not sure how to force the browser to exit.
Ctrl + C does not seem to be working.
How can I force cog to exit with the keyboard?

**Comment by philn** [comment]
It seems this is not implemented yet. Might be worth moving this discussion to a new issue...
cc @aperezdc

**Reply by chrisvollorenew** [reply]
@philn thank you for confirming this! I look forward to hearing when you implement this. In the meantime, I will explore other ways to achieve this functionality for my application.

**Reply by philn** [reply]
https://github.com/Igalia/cog/issues/495
